src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/hiddenfield/Example7.java

```diff
@@ -7,22 +7,18 @@
   </module>
 </module>
 */
+//non-compiled with javac: Compilable with Java17
+package com.puppycrawl.tools.checkstyle.checks.coding.hiddenfield;
 
 // xdoc section -- start
-abstract class SomeClass {
+abstract class Example7 {
```

Comment:
Please make Example7 to be same as all other Examples by structure.
You can add abstract method to all of them.

Reply:
Done.